lambda-s3-translate-cdk/pattern/lambda_s3_translate_cdk_stack.py

```diff
@@ -0,0 +1,64 @@
+
+class LambdaS3TranslateCdkStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        #Input Bucket
+        self.input_bucket = s3.Bucket(self, "input-bucket",
+                            versioned=True,
+                            encryption=s3.BucketEncryption.S3_MANAGED,
+                            block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
+                            enforce_ssl=True)
+
+        # Destination Bucket
+        self.destination_bucket = s3.Bucket(self, 'destination-bucket',
+                                versioned=True,
+                                encryption=s3.BucketEncryption.S3_MANAGED,
+                                block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
+                                enforce_ssl=True)
+        
+        python_lambda_kwargs = {
+            'handler': 'lambda_function.lambda_handler',
+            'runtime': lambda_.Runtime.PYTHON_3_9,
+            'timeout': Duration.minutes(2)
+        }
+
+        # Trigger Transcription Lambda
+        trigger_translate_lambda = lambda_.Function(self, "file-upload-trigger", **python_lambda_kwargs,
+                                                        code=lambda_.Code.from_asset(
+                                                            './pattern/assets'),
+                                                        function_name="start-translate",
+                                                        initial_policy=[
+                                                            iam.PolicyStatement(
+                                                                actions=['translate:TranslateText',
+                                                                         'comprehend:DetectDominantLanguage'
+                                                                         ],
+                                                                resources=['*']),
+                                                            iam.PolicyStatement(actions=['s3:PutObject'],
+                                                                                resources=[
+                                                                                    self.destination_bucket.bucket_arn+'/*']
+                                                                                ),
+                                                            iam.PolicyStatement(actions=['s3:GetObject',
+                                                                                        's3:ListBucket'],
+                                                                                resources=[
+                                                                                    self.input_bucket.bucket_arn,
+                                                                                    self.input_bucket.bucket_arn + '/*'
```

Comment:
Whats with the extra space here in the file formatting? Is this normal?

Reply:
yes, That extra space doesn't really make any change. a + b is same as a+b in python